ar-eg/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="ar">
+<head>
+  <title>ماتطلبش انك تسأل، اسأل على طول.</title>
+  <meta property="og:title" content="ماتطلبش انك تسأل، اسأل على طول." />
+  <meta property="og:type" content="website" />
+  <meta property="og:url" content="https://dontasktoask.com/ar-eg" />
+  <meta property="og:locale" content="ar_EG" />
+  <meta property="og:image" content="https://dontasktoask.com/favicon.png"/>
+  <link rel="icon" href="/favicon.png" type="image/png" />
+  <link rel="stylesheet" type="text/css" href="../style.css">
+  <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Open+Sans">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <style>
+      blockquote{
+        border-right: 10px solid #CCC;
+        border-left: initial;
+      }
+      .name{
+          display: block;
+      }
+  </style>
+</head>
+<body dir="rtl">
+  <main>
+    <h1>ماتطلبش انك تسأل، اسأل على طول.</h1>
+    <p>
+        من وقت للتاني، بيجي حد ويقول:
+    </p>
+    <blockquote>
+      <span class="name">فولان:</span>
+      <p class="message">
+        حد يعرف جافا هنا؟؟
+      </p>
+    </blockquote>
+    <p>
+        الطريقة دي وحشة جدا في السؤال لاكتر من سبب. هو الشخص ده عايز يقول:
+    </p>
+    <blockquote>
+      <span class="name">فولان:</span>
+      <p class="message">
+          حد يعرف جافا هنا يساعدني في المشكلة بتاعتي اللي ممكن تطلع ملهاش علاقة بجافا اصلا او لو حد مش عارف جافا ممكن يجاوب على سؤالي؟
+      </p>
+    </blockquote>
+    <p>
+        في اسباب كتير ممكن تخلي الناس اللي عندهم خبرة مايجوبوش على سؤالك، سؤالك بيطلب اكتر ما انت فاكر.
+    </p>
+    <p>
+        انت كده بتحمل الناس مسئولية، اكنك بتختبر ثقتهم في قدراتهم.
+        
+        وكده غصب عنك انت بتخلي ناس ماتجاوبش على سؤالك، يعني انا ممكن لو فيه سؤال عن لغة برمجة انا معرفش عنها حاجة اقدر اجاوب عليه بالـ"سينس" كده.
```

Comment:
>سينس 
>
can you explain this word ?

Reply:
Sometimes we use english words in our talk, so it is `Sense` 

So for example:

> I solve the problem by sense

> بحل المشكلة بالـسينس